.github/workflows/e2e.yml

```
@@ -15,12 +15,19 @@ jobs:
         deploytool: ['operator', 'helm']
         globalnet: ['', 'globalnet']
         cable_driver: ['libreswan', 'strongswan', 'wireguard']
+        ovn: [ '', 'ovn' ]
         exclude:
           # Our Helm setup doesn’t know how to deploy other cable drivers
           - deploytool: 'helm'
             cable_driver: 'wireguard'
           - deploytool: 'helm'
             cable_driver: 'libreswan'
+          - ovn: 'ovn'
+            deploytool: 'helm'
+          - ovn: 'ovn'
+            globalnet: 'globalnet'
+          - ovn: 'ovn'
+            cable_driver: 'wireguard'
```

```suggestion
          - ovn: 'ovn'
            cable_driver: 'wireguard'
```

We shall test this, let's skip strongswan, we should probably start removing it.

---

@mangelajo did you mean cable_driver: 'strongswan'?

---

Looks like it didn't pick OVN in the OVN job:
https://github.com/submariner-io/submariner/pull/1082/checks?check_run_id=1694249318